instrumentation/spring/spring-rmi-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/springrmi/client/ClientInstrumentationModule.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.springrmi.client;
+
+import static java.util.Collections.singletonList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class ClientInstrumentationModule extends InstrumentationModule {
+
+  public ClientInstrumentationModule() {
+    super("spring-rmi", "spring-rmi-4.0", "spring-rmi-client");
```

Comment:
I know that the base RMI instrumentation has several separate instrumentation modules for every part, but honestly I'm not sure whether this is a good idea in general - WDYT about keeping a single `InstrumentationModule` with all the instrumentations here?

Reply:
In other instrumentations I saw the Client and Server `InstrumentationModule` are usually kept separate, and there's merit in that. But we can also have all of them in a single module, I'm fine with both.

Reply:
Usually we try to have a single `InstrumentationModule` for a single library that we instrument - this makes the runtime reference matching process a bit more efficient; and makes it harder to just partially instrument the code (usually resulting in invalid telemetry). There are some instrumentations that do not follow that rule, usually because they consist of features that need to be turned on/off separately -- but the general recommendation is to keep everything in a single `InstrumentationModule`.

Reply:
Got it, thanks for the explanation.
I'll transfer all the instrumentation in a single `InstrumentationModule` and push a commit.